Delicate-Butterflies/Bangazon-Site
# Sb add product

Related Issue: #1 and #10

Description: User can add a product to sell. That product also features the product type and has input validation.

Chai/Mocha Tests File:

PR Testing Steps:
- ```git fetch``` the branch
- npm install (express-validator package added)
- npm run dbrb (optional step if you want a clean db)
- login as an existing user or create a user
- go to the 'Sell a Product' link
- try the form with different inputs
- be sure to use this image for the URL http://data.whicdn.com/images/52410577/large.jpg (MANDATORY STEP)
- verify your product has been added when you are redirected to the product view
- mission complete

Check list before you make a PR:
